src/BTVNanoCommissioning/utils/correction.py

```diff
@@ -117,8 +117,6 @@ def load_jmefactory(campaign, path):
     with importlib.resources.path(_jet_path, path) as filename:
         with gzip.open(filename) as fin:
             jmestuff = cloudpickle.load(fin)
-
-        # jet_factory = jmestuff["jet_factory"]
```

Comment:
Hi @Ming-Yan, just noticed now: the function `load_jmefactory` looks almost identical to the function `load_metfactory`, see [here](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/ea15ce3204499c5c24c51be72f408479c60dc549/src/BTVNanoCommissioning/utils/correction.py#L134), what is the difference between them?

Reply:
`load_jmefactory` includes both `jet_factory` and `met_factory` while `load_metfactory` or load_jetfactory` only loads one of them